vignettes/Introduction_To_BiocParallel.Rmd

```diff
@@ -0,0 +1,961 @@
+---
+title: "**Introduction to *BiocParallel***"
+author: "Phyllis Atieno"
```

Comment:
I do not think it is correct for you to claim sole authorship of this vignette. The material was developed by others, and they deserve (primary) credit

Reply:
I've created a commit  'Added authors" that solves this.